docs/updating.rst

```diff
@@ -2,35 +2,56 @@
 How to update the Atomic Developer Bundle to the latest version
 ===============================================================
 
-* If you are using the image from atlas.hashicorp.com
+Depending on your implementation of the ADB, you might be able to directly
+update your Vagrant box. In some cases, you will need to perform a clean
+installation or follow additional steps to complete the update.
 
-  1. Check if a newer version of the box is available use this command::
+--------------------------------------------------
+Update the ADB for images from atlas.hashicorp.com
+--------------------------------------------------
 
-      # vagrant box outdated
+1. Check if a newer version of the box is available use this command::
 
-  2. If a newer version is available, the output should look like this::
+    # vagrant box outdated
 
-        # vagrant box outdated
+2. If a newer version is available, the output should look like this::
 
-     Checking if box 'projectatomic/adb' is up to date...
-     A newer version of the box 'projectatomic/adb-testing' is available! You
-     currently have version '1.3.1'. The latest is version '1.3.2'. Run ``vagrant
-     box update`` to update.
+    # vagrant box outdated
 
-  3. If you see the above output, update the ADB Vagrant image with this
-     command::
+   Checking if box 'projectatomic/adb' is up to date...
```

Comment:
@thatdocslady I think the lines from 21 to 24 should be in grey color as it is output of the `vagrant box outdated` command.